### ISSUE TITLE
Display required Starcounter version

### ISSUE THREAD

**Issue by warpech** [Polyjuice/AppStoreClient#19]
In `project.config` of all applications, I've added the version of Starcounter that is required by the app.

Previously this information was part of the description string, which was not very useful.

I propose to display it in the AppStoreClient using a Font Awesome icon, such as:

<img width="112" alt="screenshot 2015-08-13 17 00 58" src="https://cloud.githubusercontent.com/assets/566463/9253204/e4c30e9e-41dc-11e5-9a76-0f67c1e4493d.png">

Green is if the currently installed Starcoutner version is equal or higher than required by the app.
Orange if we cannot decide.
Red if the currently installed Starcounter version is older than required.

@alemoi, @miyconst let's discuss it before implementing.

**Comment by miyconst** [comment]
Good idea. In the AppStoreClient we have free left bottom corner of each app, the icon could fit in there just fine.

The question is how to make sure, that the app is compatible with a newer Starcounter build?

I can remember lots of breaking changes in Starcounter. It would be really frustrating to see a green check icon with a non-working app.
